dask/dataframe/io/tests/test_sql.py

```diff
@@ -35,11 +37,11 @@ def db():
 
 def test_simple(db):
     # single chunk
-    data = read_sql_table('test', db, npartitions=2, index_col='number'
+    data = read_sql_table(u'test', db, npartitions=2, index_col=u'number'
```

Comment:
You should be able to drop the `u` here.  All text literals are now unicode in python 2